data/human/navy missions intro.txt

```diff
@@ -0,0 +1,1496 @@
+			choice
+				`	"Sure, what do I have to do?"`
+				`	"Sorry, I'm a bit busy right now."`
+					defer
+			`	"A ship named the <npc> has captured a number of wealthy merchants and is holding them for ransom. While this should not be a job for an ordinary contractor, there are concerns that they might kill the hostages if approached by a ship flying Republic colors. You will disable the ship and return to me immediately."`
+			choice
+				`	"Consider it done."`
+				`	"I'm sorry, but I can't deal with problems like that."`
+					decline
+			`	Captain Shah gives a small, relieved nod. "The ship was last spotted in Tais. We believe it is still there."`
+				accept
+	npc disable
+		personality heroic staying nemesis target
+		government Bounty
+		system "Tais"
+		ship "Falcon (Heavy)" "Gediminas"
+		dialog `The <npc> has been disabled. You can claim the bounty payment by returning to <destination>.`
+
+	on accept
+		log `Hunting a pirate ship called the "<npc>," which has captured multiple merchant captains. The ship was last sighted in the Tais system.`
+
+	on complete
+		"assisted republic" ++
+		payment 250000
+		dialog `Soon after you land, Captain Shah contacts you. "The hostages were recovered safely," she says, with a smile. "The Republic owes you a debt of gratitude. If you are interested in more tangible benefits, <payment> have been transferred to your account, and I have informed my superiors that you can be counted on."`
+		log "Captain Shah says she has informed her superiors of my contributions."
+
+
+
+mission "Republic Scouting 1"
+	name "Hyperspace Relay Testing"
+	description "Fly through the <waypoints> systems, then report to <destination> so the Navy can analyze the network connectivity of your logs."
+	minor
+	waypoint "Hassaleh"
+	waypoint "Sumar"
+	waypoint "Cardax"
+	waypoint "Volax"
+	source
+		government "Republic"
+		attributes "paradise"
+	destination "Glaze"
+	to offer
+		has "event: war begins"
+		not "chosen sides"
+		random < 60
+
+	on offer
+		conversation
+			`As you wander the spaceport you overhear a woman in a Navy uniform telling a bored-looking merchant about satellite dishes in the empty systems to the north.`
+			`	"That sounds interesting." The merchant tells her, "But I've got a cargo bay full of medical supplies headed the other direction."`
+			`	"Also an important mission," she replies. "I wish you safe travels."`
+			choice
+				`	(Ask her what's going on.)`
+				`	(Walk away.)`
+					decline
+			`	"Do you need help with retrieving a satellite?" you ask the Navy officer, interested, as the other merchant captain walks away.`
+			`	"Not quite." She replies brightly. "We've been experiencing some latency between Farpoint and Maelstrom. I'd need someone to fly through, then let me look at their logs so I can figure out what the trouble is."`
```

Comment:
Canonically there are no hyperlane communication links through this part of space.

Reply:
In FW Reconciliation you get this
https://github.com/endless-sky/endless-sky/blob/c1afeac04f64757780dce6c0a18ecfe5d7777534/data/human/free%20worlds%20reconciliation.txt#L1057
> "all communications links with Syndicate space have been cut. The Net relies on hyperspace links to transmit messages, and there are no transmitter relays in the uninhabited systems that now form the only remaining link to Syndicate space."
The whole of human space is linked, but not through those systems.

Reply:
I was pointed to this reply, and asked to give a little feedback on it. 

> In FW Reconciliation you get this
> https://github.com/endless-sky/endless-sky/blob/c1afeac04f64757780dce6c0a18ecfe5d7777534/data/human/free%20worlds%20reconciliation.txt#L1057
> 
> The whole of human space is linked, but not through those systems.

All of human space is linked except when the plot demands it isn't. There is no reason why Maelstrom and Farpoint would not have some form of a hyperspace relay network between the two, especially as it is heavily implied that Farpoint was made into such a fortress world due to first contact with the Hai (Yet Maelstrom isnt for some reason?) So of course there would likely be a redundant relay for managing some sensor equipment, and keeping in loose contact with human patrols around hai space. 

This is something the Moon and I took into great consideration during the planning of these missions. We decided on the more likely story that the relays in these systems were destroyed or otherwise jammed by the Pug. We also considered that perhaps Moktar was the break point. In the end, we decided that the actual details didn't matter so long as they didnt create a massive plot hole, which neither of these solutions would. In both events, Syndicate space would still be cut off no matter which way it was spun.